doc/overlapping.md

```diff
@@ -0,0 +1,145 @@
+# Overlapping strategy
```

Comment:
it would be helpful to add a short sentence saying something like "this document covers the logic behind overlapping strategy (called concentrated liquidity in the UI). A concentrated liquidity consists of two orders that have an overlapping price range, usually for the purpose of... main terminology consists of ... high/low, market price, spread..." etc.